tests/framework/e2e/downgrade.go

```diff
@@ -157,6 +157,13 @@ func DowngradeUpgradeMembersByID(t *testing.T, lg *zap.Logger, clus *EtcdProcess
 		if err := member.Stop(); err != nil {
 			return err
 		}
+
+		// When we downgrade or upgrade a member, we need to re-generate the flags, to convert some non-experimental
+		// flags to experimental flags, or vice verse.
+		if err := clus.UpdateProcOptions(memberID, t); err != nil {
```

Comment:
This will not be called in robustness tests

Reply:
My understanding is both robustness test and e2e test reuse the same `DowngradeUpgradeMembersByID` to perform downgrade & upgrade.

If it doesn't, then robustness test will fail.

But the robustness test on older version is periodically test, can't be verified immediately. So I raised https://github.com/kubernetes/test-infra/pull/34834

Reply:
ack